src/NuGet.Services.EndToEnd/NuGetExeTests.cs

```diff
@@ -124,6 +124,29 @@ public async Task LatestNuGetExeCanVerifyRepositorySignedPackage(PackageType pac
                 hasRepositorySignature: _testSettings.IsRepositorySigningEnabled);
         }
 
+        [Theory]
+        [MemberData(nameof(PackageAndSourceTypes))]
+        public async Task Pre490NuGetExeCanVerifyRepositorySignedPackage(PackageType packageType, PackageType[] dependencies, bool semVer2, SourceType sourceType)
```

Comment:
The 4.9.0 repository signing resource is verified by the `LatestNuGetExeCanVerifyRepositorySignedPackage` test